sw/airborne/modules/multi/tcas.c

```diff
@@ -219,22 +236,24 @@ void tcas_periodic_task_1Hz(void)
 void tcas_periodic_task_4Hz(void)
 {
   // set alt setpoint
-  if (stateGetPositionUtm_f()->alt > GROUND_ALT + SECURITY_HEIGHT && tcas_status == TCAS_RA) {
-    struct ac_info_ * ac = get_ac_info(tcas_ac_RA);
+  if (stateGetPositionUtm_f()->alt > SECURITY_ALT && tcas_status == TCAS_RA) {
+    struct EnuCoor_f *ac = acInfoGetPositionEnu_f(tcas_ac_RA);
     switch (tcas_resolve) {
       case RA_CLIMB :
-        tcas_alt_setpoint = Max(nav_altitude, ac->alt + tcas_alim);
+        tcas_alt_setpoint = Max(nav_altitude, ac->z + tcas_alim);
         break;
       case RA_DESCEND :
-        tcas_alt_setpoint = Min(nav_altitude, ac->alt - tcas_alim);
+        tcas_alt_setpoint = Min(nav_altitude, ac->z - tcas_alim);
         break;
       case RA_LEVEL :
       case RA_NONE :
         tcas_alt_setpoint = nav_altitude;
         break;
+      default:
+        break;
     }
     // Bound alt
-    tcas_alt_setpoint = Max(GROUND_ALT + SECURITY_HEIGHT, tcas_alt_setpoint);
```

Comment:
wonder if we shouldn't rather use `ground_alt + SECURITY_HEIGHT` so it takes the geo init changes into account...